### ISSUE TITLE
VR ARC Stem Method Output Incorrect

### ISSUE THREAD

**Issue by naijierubia** [tryreplay/replay-issues#25]
When I used the VR ARC Stem Method model to separate vocals and instruments, I found that the output vocals and instrument files were incorrect. The human voice used for converting music is actually an instrument, while the instrument is actually a human voice, So the final result was a terrible one.I changed the song and cleared the cache, but the same error still occurred.And using UVR5 for separation will not result in such errors

[(converted-vocals) unknown sings Nai Br.XX,Celeina Ann - Kiss Me-2023-12-20.zip](https://github.com/tryreplay/replay-issues/files/13723939/converted-vocals.unknown.sings.Nai.Br.XX.Celeina.Ann.-.Kiss.Me-2023-12-20.zip)

**Comment by jonluca** [comment]
Which one are you using? Karaoke?

**Comment by jonluca** [comment]
Can you try v3.3.0

**Comment by naijierubia** [comment]
> Can you try v3.3.0
Yes, I use Karaoke. However, after I updated to V3.3.0, the output using this model all changed to current sound
